redaxo/src/core/backend.php

```diff
@@ -79,10 +79,10 @@
         $login->setLogout(true);
         $login->checkLogin();
         rex_csrf_token::removeAll();
-        rex_response::setHeader('Clear-Site-Data', '"cache", "cookies", "storage", "executionContexts"');
+        rex_response::setHeader('Clear-Site-Data', '"cache", "storage", "executionContexts"');
 
-        // Currently browsers like Safari do not support the header Clear-Site-Data. This ensures that all session variables are still removed.
-        session_destroy();
+        // we dont kill/regenerate the session so e.g. the frontend will not get logged out
+        rex_request::clearSession();
```

Comment:
Es gibt hier in der backend.php noch eine weitere Stelle, wo `session_destroy` verwendet wird. Die muss vermutlich auch angepasst werden, oder?

https://github.com/redaxo/redaxo/commit/0da602b1111d79b39bca3b3a357a6dbd81cdd70c

Reply:
Jop

Reply:
Fixed